generator/typescript/dropbox_types.d.ts

```diff
@@ -29,7 +29,7 @@ declare module DropboxTypes {
      * @param state State that will be returned in the redirect URL to help
      *   prevent cross site scripting attacks.
      */
-    getAuthenticationUrl(redirectUri: string, state?: string, authType = 'token'): string;
+    getAuthenticationUrl(redirectUri: string, state?: string, authType?: 'token' | 'code'): string;
```

Comment:
This fixes https://github.com/dropbox/dropbox-sdk-js/issues/199

Reply:
I'm afraid this will rollback to the wrong code when I generate the SDK again with a stone/spec change. Let's keep the change, and I'll fix the generator stuff before the next update.

Reply:
👍 Also, happy to help with fixing the generator code if you can point me in the right direction